packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -173,7 +158,13 @@ const ChooseOfferForm: React.FC<Props> = ({
           </label>
         </div>
       )}
-      {<div className={styles.offers}>{!offers.length ? <p>{t('choose_offer.no_pricing_available')}</p> : offers.map(renderOfferBox)}</div>}
+      <div className={styles.offers}>
+        {!offers.length ? (
+          <p>{t('choose_offer.no_pricing_available')}</p>
+        ) : (
+          offers.map((offer) => <OfferBox key={offer.offerId} offer={offer} values={values} onChange={onChange} />)
+        )}
```

Comment:
Just slightly more neat: :-) 

```suggestion
          offers.map((offer) => <OfferBox key={offer.offerId} offer={offer} selected={offer.offerId === values.offerId} onChange={onChange} />)
        )}
```

Reply:
Ah, yes of course! Well spotted ;) I admire that you found this optimisation